nemo/utils/exp_manager.py

```diff
@@ -1122,6 +888,8 @@ def configure_checkpointing(
     if 'mp_rank' in checkpoint_callback.last_model_path or 'tp_rank' in checkpoint_callback.last_model_path:
         checkpoint_callback.last_model_path = uninject_model_parallel_rank(checkpoint_callback.last_model_path)
     trainer.callbacks.append(checkpoint_callback)
+    preemption_callback = PreemptionCallback(checkpoint_callback)
```

Comment:
Will this be a default ? I would suggest to add a bool flag that enables by default but can be disabled if wanted.

Reply:
Makes sense, changed this in the latest commit